ntoskrnl/ex/handle.c

```diff
@@ -18,6 +18,7 @@
 LIST_ENTRY HandleTableListHead;
 EX_PUSH_LOCK HandleTableListLock;
 #define SizeOfHandle(x) (sizeof(HANDLE) * (x))
```

Comment:
Is "SizeOfHandle" still used in the code? (since I see many replacements of SizeOfHandle to INDEX_TO_HANDLE_VALUE)